packages/components/src/templates/next/layouts/Collection/CollectionResults.tsx

```diff
@@ -54,17 +81,27 @@ export const CollectionResults = ({
         </div>
       </div>
       {/* NOTE: DO NOT add h-full to this div as it will break old browsers */}
-      <div className="flex w-full flex-col gap-0">
+      <div className={collectionResults()}>
         {paginatedItems.length > 0 &&
-          paginatedItems.map((item) => (
-            <CollectionCard
-              key={Math.random()}
-              {...item}
-              shouldShowDate={shouldShowDate}
-              siteAssetsBaseUrl={siteAssetsBaseUrl}
-              LinkComponent={LinkComponent}
-            />
-          ))}
+          paginatedItems.map((item) =>
+            variant === "collection" ? (
+              <CollectionCard
+                key={Math.random()}
```

Comment:
hmm using math.random for keys is not a good practice since React cannot identify which nodes have changed during reconciliation, can use any other "id"?

Reply:
hmm, that's true - swapped over to using `title + category` since they are both required fields but that runs the risk of duplicaet ids (albeit very minor)